assets/js/apps/search/components/filters/category-filter/CategoryFilter.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import CategoryFilter  from './CategoryFilter';
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before 'from'.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
import CategoryFilter from './CategoryFilter';
```